etc/env/base/database.toml

```diff
@@ -1,16 +1,17 @@
+sqlx_offline = true
```

Comment:
Please add a comment mentioning that we intentionally want the workspace to use offline mode by default and need to run `zk db prepare` to update `sqlx` files.